docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -88,7 +93,19 @@ Complete the **G-Flags** section as follows:
 
   - [YB T-Server Configuration Flags](/preview/reference/configuration/yb-tserver/#configuration-flags)
 
-Accept default values for all of the remaining fields.
+Optionally, use the **Helm Overrides** section as follows:
+
+- Click **Add Kubernetes Overrides** to open the **Kubernetes Overrides** dialog shown in the following illustration:
+
+  ![img](/images/yb-platform/kubernetes-config66.png)
+
+- Using the sample format, specify the universe-level overrides for YB-Master and YB-TServer, as well as availability zone overrides.
+
+- Select **Force Apply** if you want to override any previous overrides.
+
+- Click **Validate & Save**. 
+
+  If there are any errors in your overrides definitions, a detailed error message is displayed. You can correct the errors and try to save again.
```

Comment:
The force apply is used to override any validation errors. So if there are validation errors but you want to go ahead and save, you would use this checkbox

Reply:
Fixed